src/lib/utils.ts

```diff
@@ -96,7 +97,7 @@ export function timestampFromDateNanos(
   const nanos = (date.getTime() % 1000) * 1000000 + (date.nanoseconds ?? 0);
   return Timestamp.fromPartial({
     seconds: new Long(date.getTime() / 1000),
-    nanos,
```

Comment:
Lots of small changes from your editor/prettier config.

Can you run `yarn fix` to bring this into a [standard format](https://github.com/confio/ts-relayer/blob/main/package.json#L10)? (Maybe we need to add an explicit prettier config for the editors.